### PR TITLE
*: Add platform owners

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Adding platform owners helps increase the people should help reduce bottleneck on `installer-approvers`

/assign @sdodson 
/cc @openshift/openshift-team-installer 
/cc @stbenjam @hardys

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -25,13 +26,31 @@ aliases:
...
+    - hardys
+    - stbenjam
   baremetal-reviewers:
     - celebdor
     - markmc
     - russellb
     - hardys
     - stbenjam
+  aws-approvers:
+    - patrickdillon
```
Comment (sdodson) [inline OWNERS_ALIASES]: If we drop auto approval we'll need to add a second here so PRs from @patrickdillon aren't blocked.

**Comment by sdodson** [comment]
/approve
I'm definitely in favor of federating review and approval.
What's the behavior of empty reviewers aliases? Does that just default back to allowing lgtm from organization members?

**Comment by abhinavdahiya** [comment]
> What's the behavior of empty reviewers aliases? Does that just default back to allowing lgtm from organization members?
> https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners

```
reviewers: a list of GitHub usernames or aliases that are good candidates to /lgtm a PR
```

it is used to find good candidates, and doesn't affect actually who can lgtm.

File: OWNERS_ALIASES (hunk 1 above)
Comment (abhinavdahiya) [reply OWNERS_ALIASES]: well i'm thinking we should use the <platform>-approvers as additive to installer-approvers to reduce duplication...

**Pull request merged**